fitbenchmarking/fitbenchmarking
# 825 matlab controller mixin

#### Description of Work

Fixes #825 


#### Testing Instructions

1. Test that matlab controllers still run as expected
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
